lib/improver/tests/set_up_test_cubes.py

```diff
@@ -40,6 +40,7 @@
 import numpy as np
 import iris
 from iris.coords import DimCoord
+from cartopy.crs import PlateCarree
```

Comment:
Sorry, my request was bad, hence the failure in load cube. This needs to be an iris coordinate system not cartopy, as is the case in the STANDARD_GRID_CCRS/